presto-main/src/test/java/com/facebook/presto/sql/analyzer/TestMaterializedViewQueryOptimizer.java

```diff
@@ -125,6 +125,11 @@ public void testWithDerivedFields()
         expectedRewrittenSql = format("SELECT SUM(mv_sum) as sum_of_abc, MAX(mv_max) as max_of_abc, mv_d, e FROM %s GROUP BY mv_d, e", VIEW);
 
         assertOptimizedQuery(originalViewSql, baseQuerySql, expectedRewrittenSql);
+
+        originalViewSql = format("SELECT SUM(a * b + c) as mv_sum, MAX(a * b + c), d as mv_d, e FROM %s GROUP BY d, e", BASE_TABLE_1);
```

Comment:
I was trying to check if the function call column mv definition has an alias, in case it would be rewritten correctly:
For example:
For a MV ```CREATE MATERIALIZED VIEW mv SELECT SUM(a), b FROM base GROUP BY b```.
The query ```SELECT SUM(a), b FROM base GROUP BY b``` should not be rewritten as ```SELECT SUM(SUM(a)), b FROM mv GROUP BY b```

Let me remove this test case as long as we can guarantee that no materialized view has columns like ```SUM(a)``` without an alias.